### ISSUE TITLE
Replace selected icon with svg

### ISSUE THREAD

**Issue by edenizk** [edenizk/grapesjs-google-material-icons#3]
When you activate it should replace selected element with SVG element

**Comment by edenizk** [comment]
On hold until someone requests

**Comment by micker** [comment]
yes it can be cool or adding more selected icon library (like iconify fontanswone .. etc) ?

**Comment by edenizk** [comment]
Hi @micker,

yes that sounds like a cool idea, I will consider about it and work on it if I will have some free time 🙂
I've added a task for it #22

**Comment by micker** [comment]
Great rhanks
